docs/core/tools/dotnet-nuget-sign.md

```diff
@@ -0,0 +1,162 @@
+
+`dotnet nuget sign` - Signs all the NuGet packages matching the first argument with a certificate.
+
+## Synopsis
+
+```dotnetcli
+dotnet nuget sign [<package-path(s)>]
+    [-o|--output <OUTPUT DIRECTORY>]
+    [--certificate-path <PATH>]
+    [--certificate-store-name <STORENAME>]
+    [--certificate-store-location <STORELOCATION>]
+    [--certificate-subject-name <SUBJECTNAME>]
+    [--certificate-fingerprint <FINGERPRINT>]
+    [--certificate-password <PASSWORD>]
+    [--hash-algorithm <HASHALGORITHM>]
+    [--timestamper <TIMESTAMPINGSERVER>]
+    [--timestamp-hash-algorithm <HASHALGORITHM>]
+    [--overwrite]
+    [-v|--verbosity <LEVEL>]
+
+dotnet nuget sign -h|--help
+```
+
+## Description
+
+The `dotnet nuget sign` command signs all the packages matching the first argument with a certificate. The certificate with the private key can be obtained from a file or from a certificate installed in a certificate store by providing a subject name or a SHA-1 fingerprint.
+
+## Arguments
+
+- **`package-path(s)`**
+
+  Specifies the file path to the package(s) to be signed. Multiple position arguments can be passed in to verify multiple packages.
+
+## Options
+
+- **`-o|--output`**
+
+  Specifies the directory where the signed package should be saved. If this option is not specified, by default the original package is overwritten by the signed package.
+
+- **`--certificate-path <PATH>`**
+
+  Specifies the file path to the certificate to be used in signing the package.
+
+> [!NOTE]
+> This command currently supports only `PKCS12 (PFX)` files that contain the certificate's private key.
+
+- **`--certificate-store-name <STORENAME>`**
+
+  Specifies the name of the X.509 certificate store to use to search for the certificate. Defaults to "My", the X.509 certificate store for personal certificates. This option should be used when specifying the certificate via --certificate-subject-name or --certificate-fingerprint options.
+
+- **`--certificate-store-location <STORELOCATION>`**
+
+  Specifies the name of the X.509 certificate store use to search for the certificate. Defaults to "CurrentUser", the X.509 certificate store used by the current user. This option should be used when specifying the certificate via --certificate-subject-name or --certificate-fingerprint options.
+
+- **`--certificate-subject-name <SUBJECTNAME>`**
+
+  Specifies the subject name of the certificate used to search a local certificate store for the certificate. The search is a case-insensitive string comparison using the supplied value, which will find all certificates with the subject name containing that string, regardless of other subject values. The certificate store can be specified by --certificate-store-name and --certificate-store-location options.
+
+> [!NOTE]
+> This command currently supports only a single matching certificate in the result. If there are multiple matching certificates in the result, or no matching certificate in the result, the command will fail.
```

Comment:
```suggestion
  > [!NOTE]
  > This command currently supports only a single matching certificate in the result. If there are multiple matching 
certificates in the result, or no matching certificate in the result, the command will fail.
```

Reply:
"This command" refers to what? to "--certificate-subject-name"? If so, would "switch" be more suitable than "command" here?

Reply:
Thanks! Changed into "option".